bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/exceptions/RuntimeErrors.java

```diff
@@ -66,6 +66,7 @@
     CYCLIC_VALUE_REFERENCE("cyclic.value.reference", "RUNTIME_0037"),
     CANNOT_CONVERT_NIL("cannot.convert.nil", "RUNTIME_0038"),
     INCOMPATIBLE_CONVERT_OPERATION("incompatible.convert.operation", "RUNTIME_0039"),
+    INCOMPATIBLE_CONVERT_OPERATION_AMBIGUOUS_TARGET("ambiguous.target.convert.operation", "RUNTIME_0096"),
     INCOMPATIBLE_SIMPLE_TYPE_CONVERT_OPERATION("incompatible.simple.type.convert.operation", "RUNTIME_0040"),
     TUPLE_INDEX_OUT_OF_RANGE("tuple.index.out.of.range", "RUNTIME_0038"),
```

Comment:
Shall we refactor this file such that there are no duplicate error codes (currently "RUNTIME_0038" is used twice in line 71 and 67) and it is easy to add new or remove new errors. I was thinking of arranging errors in the ascending order of error codes here and bundle the same kinds of errors in "MessagesBundle.properties" file. In future if we want to delete an error we delete it here keeping the space in this file and not in the other file. If we want to add error we add to the first empty space with the relevant code here and add adjacent to the same kinds in "MessagesBundle.properties" file. Or use a better way.